indico/modules/events/abstracts/forms.py

```diff
@@ -390,30 +390,38 @@ class CreateEmailTemplateForm(EditEmailTemplateRuleForm):
                      "You can customize it later."))
 
 
-class AbstractForm(IndicoForm):
+class AbstractAttachmentForm(IndicoForm):
+    attachments = FileField(_('Attachments'), param_name='attachments', multiple_files=True)
+
+    def __init__(self, *args, **kwargs):
+        self.abstract = kwargs.pop('abstract', None)
+        self.event = kwargs.pop('event')
+        super(AbstractAttachmentForm, self).__init__(*args, **kwargs)
+
+
+class AbstractForm(AbstractAttachmentForm):
+    _order = ('title', 'description', 'submitted_contrib_type', 'person_links', 'attachments', 'submission_comments')
```

Comment:
:warning: Where are the track-related fields from the single/multi track mixins?